wme-sitebuilder/Wizards/GoLive.php

```diff
@@ -128,6 +179,194 @@ public function verifyDomain() {
 		}
 	}
 
+	/**
+	 * AJAX: search domains.
+	 */
+	public function searchDomains() {
+		if ( ! current_user_can( 'manage_options' ) ) {
+			return wp_send_json_error( new WP_Error(
+				'mapps-capabilities-failure',
+				__( 'You do not have permission to perform this action. Please contact a site administrator or log into the Nexcess portal to change the site domain.', 'nexcess-mapps' )
+			), 403 );
+		}
+
+		// Verify the domain structure.
+		$domain = ! empty( $_POST['domain'] )
+			? Domain::parseDomain( $_POST['domain'] )
+			: null;
+
+		$domain = $this->domains->formatDomain( $domain );
+
+		if ( empty( $domain ) ) {
+			return wp_send_json_error( new WP_Error(
+				'mapps-invalid-domain',
+				sprintf(
+					/* Translators: %1$s is the provided domain name. */
+					__( '"%s" is not a valid domain name. Please check your spelling and try again.', 'nexcess-mapps' ),
+					sanitize_text_field( $_POST['domain'] )
+				)
+			), 422 );
```

Comment:
Wrong translation tag and remove the reference to mapps and replace with something WME specific.